api/services/Flairs.js

```diff
@@ -261,6 +263,13 @@ exports.makeNewCSSClass = function (previous_flair, new_addition, subreddit) {
   if (new_addition === 'involvement') {
     return previous_flair.replace(/( |$)/, '1$1');
   }
+  if (extraFlair.includes(new_addition)) {
```

Comment:
Alright, well if we don't want to upgrade to 6 at the same time as this release, I'll need to move this back. I assumed the server was using babel, is it not?